unittests/CMakeLists.txt

```diff
@@ -145,11 +172,30 @@ if (NOT IS_DIRECTORY "${GTEST_INCLUDE_DIR}")
 endif()
 message(STATUS "GTEST_INCLUDE_DIR: ${GTEST_INCLUDE_DIR}")
 
+add_library(unittest_main STATIC)
```

Comment:
Any reason you added `STATIC` here?
Is this required?

Reply:
Good catch! No, I do not think it is required, so I removed it.